src/form-layout-grid.scss

```diff
@@ -192,9 +190,19 @@ $cols: 11;
 @media (min-width: 1440px) {
   .#{$blockContainer}.#{$block}-container {
     .#{$blockRow} {
+      align-items: start;
+
       .#{$blockCol} {
         .fd-form-label {
           padding-bottom: 0;
+
+          &--horizontal {
+            line-height: 2.8rem;
+          }
+
+          &--compact {
+            line-height: 2rem;
+          }
```

Comment:
These 2 modifier classes belong to the `fd-form-label` and they should bee in the corresponding scss file. Also this PR template should be updated with the Breaking changes as you are changing the HTML structure. The wiki pagee should bee updated, too.

Reply:
Done